scripts/update-image.py

```diff
@@ -456,7 +456,7 @@ def generate_plan(arch, root_uuid, scriptdir):
     yield FsAction.RSYNC, "sbin"
     yield FsAction.RSYNC, "root/"
     yield FsAction.RSYNC, "usr/"
-    yield FsAction.RSYNC, "etc/"
+    yield FsAction.RSYNC, "etc/", ["udev/hwdb.bin"]
```

Comment:
Why do we need to preserve this?

Reply:
This would get regenerated on every boot by udev otherwise, and this takes a few seconds.